woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -187,3 +197,31 @@ def test_ordinal_transform(sample_series):
 
     with pytest.raises(ValueError, match=error_msg):
         ordinal_incomplete_order.transform(sample_series)
+
+
+def test_email_address_validate(sample_df):
+    series = sample_df["email"]
+    invalid_row = pd.Series({4: "bad_email"}, name="email")
+
+    if _is_koalas_series(series):
+        invalid_row = ks.from_pandas(invalid_row)
+
+    email_address = EmailAddress()
+    assert email_address.validate(series) is None
+
+    series = series.append(invalid_row)
+    match = "Series email contains invalid email addresses."
+
+    with pytest.raises(ValueError, match=match):
+        email_address.validate(series)
+
+    actual = email_address.validate(series, return_invalid_values=True)
+
+    if _is_dask_series(actual):
+        actual = actual.compute()
+
+    if _is_koalas_series(actual):
+        actual = actual.to_frame().to_pandas()["email"]
```

Comment:
I think you can use the `to_pandas` testing utility function here and avoid checking the series type.

Reply:
updated